.gitignore

```diff
@@ -10,9 +10,7 @@ artifacts/
 build/
 cache/
 dist/
-deployments/*/*/artifacts
-deployments/*/*/cache
-deployments/*/*/contracts
+deployments/.contracts
```

Comment:
So contracts for all chains will live in the same directory? What happens if two contracts share the same address on different chains?

Reply:
I mean they can't, the addresses are deterministic

Reply:
If they share an address they are the same contract

Reply:
true